elm-pep.js

```diff
@@ -25,7 +31,13 @@ function addMouseToPointerListener(target, mouseType, pointerType) {
     let pointerEvent = new MouseEvent(pointerType, mouseEvent);
     pointerEvent.pointerId = 1;
     pointerEvent.isPrimary = true;
-    mouseEvent.target.dispatchEvent(pointerEvent);
+
+    let target = mouseEvent.target;
+    if (mouseCaptureTarget !== null) {
+      target = mouseCaptureTarget;
```

Comment:
In that case, I wonder if the `offsetX/Y` properties shouldn't also be adjusted? I cannot test it since chromium and firefox both support `setPointerCapture` from what I've tried. Could you test if safari behave coherently in this regard?

Reply:
Good point. I'm honestly a bit surprised how `setPointerCapture` seems to be implemented without the pointer events.

Reply:
Curiously, the offsetX/Y for mouse-based pointer events seems correct in Safari. I don't understand how/why.